README.md

```diff
@@ -195,6 +195,26 @@ Install custom reporters as a development dependency and you can use the `--repo
 nyc report --reporter=<custom-reporter-name>
 ```
 
+## Setting the project root directory
+
+nyc runs a lot of file system operations relative to the project root directory.
+During startup nyc will look for the *default* project root directory.
+The *default* project root directory is the first directory found that contains a `package.json` file when searching from the current working directory up.
+If nyc fails to find a directory containing a `package.json` file, it will use current working directory.
+You can change the project root directory with the `--cwd` option.
+
+**Note:** nyc stanza configuration is taken from the `package.json` file in the *default* project root directory.
+
+nyc uses the project root directory when:
+ * looking for source files to cover
+ * creating globs for include and exclude rules during file selection
+ * loading custom require hooks from the `require` array
+
+nyc may create artefact directories within the project root, such as:
```

Comment:
The Yankee in me wants to resist use of the British spelling of `artifact` :grinning: (joking, I don't really care that much)

Reply:
We've had discussions at work about this one, from my understanding, an artifact is something left behind by an ancient civilisation, while an artefact is the product of a system.  So it's not an issue of Americanisation, but a newer, less popular but more accurate word.  All that being said, Wikipedia states that it's a difference in Americanisation.

Reply:
I've never seen the word `artefact`, initially I was going to point to it as a misspelling but then decided to do a search and found that it was valid in other places.  AFAIK in American English everything left behind is an `artifact` (doesn't matter who/what left it behind).